pages/belt_docs/list.mdx

```diff
@@ -461,28 +1177,89 @@ let keepMapU: (t('a), [@bs] ('a => option('b))) => t('b);
 let keepMap: (t('a), 'a => option('b)) => t('b);
 ```
 
-## partitionU
+```reason
+keepMap(someList, f);
+```
+
+Applies `f` to each element of `someList`. If `f(xi)` returns `Some(value)`, then `value` is _kept_ in the resulting list.
```

Comment:
`xi` = `someList`?

Reply:
Same as the other one. Originally these were all called `xs` and `ys` which might have made it more clear that `f(x)` is an element of the `xs`. But I felt they could confuse, so instead settled for `someList`. Can we change the wording to make this clearer that `x` is an element of `someList`?

Reply:
Yeah I think `x` being an element of `someList` is the best approach here!